src/components/tables/TableRowWrapper.tsx

```diff
@@ -1,4 +1,13 @@
 import * as React from 'react';
 
-export const TableRowWrapper = (props?: any): JSX.Element =>
-    <tbody className={props.className}>{props.children}</tbody>;
+export interface ITableCollapsibleRowWrapperProps {
+    collapsibleRow: React.ReactNode;
+    children: React.ReactNode;
+    className?: string;
+}
+
+export const TableCollapsibleRowWrapper = (props?: ITableCollapsibleRowWrapperProps): JSX.Element =>
+    <tbody className={props.className}>
+        {props.children}
+        {props.collapsibleRow}

```

Comment:
Can't the `collapsibleRow` be just another `children` instead?

Reply:
fixed